autoload/health/nvim_treesitter.vim

```diff
@@ -0,0 +1,4 @@
+" Last Change: 2020 avril 21
```

Comment:
is this line mandatory ? it doesn't really matter since we can do an `ls -l` or a `git log` if we need this information.

Reply:
I don't really know but it is fairly standard to have those things in VimL files

Reply:
okay, i'm not really fond of adding unecessary informations inside a file but if you think you need it then keep it.